src/main.h

```diff
@@ -341,7 +341,22 @@ bool IsFinalTx(const CTransaction &tx, int nBlockHeight, int64_t nBlockTime);
  */
 bool CheckFinalTx(const CTransaction &tx, int flags = -1);
```

Comment:
IsFinalTx is planned to be part of libconsensus,   CheckFinalTx is just a convenience function using globals. I don't  consider CheckFinalTx (nor CheckSequenceLocks) a "consensus  function" or "consensus code".